src/Uno.UWP/Storage/Streams/DataWriterStoreOperation.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices.WindowsRuntime;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.Foundation;
+
+namespace Windows.Storage.Streams
+{
+	public partial class DataWriterStoreOperation : IAsyncOperation<uint>, IAsyncInfo
+	{
+		private CancellationTokenSource _cts = new CancellationTokenSource();
+		private AsyncOperationCompletedHandler<uint> _onCompleted;
+		private Task<uint> _task;
+		private AsyncStatus _status;
+		private uint _id = 0;
+		public DataWriterStoreOperation()
+		{
+
+			Status = AsyncStatus.Started;
+			//try
+			//{
+			//	AsyncInfo.Run(async ct =>
+			//	{
+
+			//	});
+			//	Status = AsyncStatus.Completed;
+			//}
+			//catch (Exception e)
+			//{
+			//	ErrorCode = e;
+			//	Status = AsyncStatus.Error;
+			//	throw;
+			//}
+		}
+
+		public AsyncOperationCompletedHandler<uint> Completed
+		{
+			get { return _onCompleted; }
+			set
+			{
+				_onCompleted = value;
+				if (Status != AsyncStatus.Started) { value?.Invoke(this, Status); }
+			}
+
+		}
+		public Exception ErrorCode { get; set; }
+		public uint Id => _id;
+		public AsyncStatus Status
+		{
+			get { return _status; }
+			set
+			{
+				_status = value;
+			}
+		}
+		public void Cancel()
```

Comment:
For one-line methods, please use the `=>` syntax (e.g. `public void Cancel() => _cts.Cancel();`